plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -193,7 +193,7 @@ fu_plugin_thunderbolt_rescan (FuPlugin *plugin, GError **error)
 		fu_device_add_guid (info->dev, guid_id);
 
 		/* format version */
-		version = g_strdup_printf ("%" G_GUINT32_FORMAT ".%02" G_GUINT32_FORMAT,
+		version = g_strdup_printf ("%" G_GINT32_MODIFIER "x.%02" G_GINT32_MODIFIER "x",
```

Comment:
This is what the documentation says for the 16-bit types https://developer.gnome.org/glib/stable/glib-Basic-Types.html#G-GINT16-MODIFIER:CAPS